insonmnia/hub/server.go

```diff
@@ -18,16 +18,14 @@ import (
 	"google.golang.org/grpc/status"
 )
 
-const (
-	// TODO: make it configurable
-	externalGRPCEndpoint         = ":10001"
-	minerHubInterconnectEndpoint = ":10002"
-)
+
 
 // Hub collects miners, send them orders to spawn containers, etc.
 type Hub struct {
 	ctx           context.Context
+	grpcEndpoint  string
```

Comment:
grpcDialer accepts target as string, because target is kind of service name (like in Cociane)